app/main.py

```diff
@@ -16,4 +16,18 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable" : [] ,
+    "immutable" : []
+}
+
+variables = [
+    lucky_number, pi, one_is_a_prime_number , name, my_favourite_films,
+    profile_info, marks, collection_of_coins
+]
+for variabl in variables:
```

Comment:
There is a typo in the variable name 'variabl'. It should be 'variable' to match the common naming convention and avoid confusion.